src/dataframe/dataframe.jl

```diff
@@ -125,6 +126,12 @@ function DataFrame(columns::AbstractVector,
     return DataFrame(convert(Vector{Any}, columns), Index(convert(Vector{Symbol}, cnames)))
 end
 
+# if #1308 is merged this sould be refactor to include makeunique
+function DataFrame(columns::AbstractMatrix, cnames::AbstractVector{Symbol} = gennames(size(columns, 2)))
```

Comment:
Better use the short function syntax for one-liners like this.

Reply:
fixed